packages/docs/docs/features/dev-tools.md

```diff
@@ -1,14 +1,14 @@
 ---
 id: dev-tools
-title: Dev Tools (experimental)
+title: Dev Tools
 sidebar_position: 7
 ---
 
-Radon IDE comes with an experimental support for launching Dev Tools as separate panels withing VSCode or Cursor.
+Radon IDE comes with support for launching Dev Tools as separate panels withing VSCode or Cursor.
 
-Currently, only a limited set of tools is available.
-The Network Inspector and Redux tools are built into the Radon IDE and don't require any additional configuration.
-The remaining tools rely on the [Expo Devtools Plugins](https://docs.expo.dev/debugging/devtools-plugins/) being setup for these individual tools to work (wee instructions below).
+The built-in tools for Network Inspector, Redux, Outline Rerenders (via [react-scan](https://react-scan.com/)) and React Query are integrated with the IDE and doesn't require any additional configuration.
+
+> NOTE: If your app is using [Expo Devtools Plugins](https://docs.expo.dev/debugging/devtools-plugins/) you will be able to use Redux and React Query dev tools via the plugin too. However, we recommend using the tools that are built into the Radon IDE instead of the one from the plugins as Radon relies on the official dev extension for Redux and React Query respecivelly.
```

Comment:
```suggestion
> NOTE: If your app uses [Expo Devtools Plugins](https://docs.expo.dev/debugging/devtools-plugins/) you will be able to use the Redux and React Query dev tools via the plugin too. However, we recommend using the tools that are built into the Radon IDE instead of the ones from the plugins as Radon relies on the official dev extension for Redux and React Query respectively.
```